lib/ansible/plugins/lookup/config.py

```diff
@@ -151,7 +151,7 @@ def run(self, terms, variables=None, **kwargs):
                 if missing == 'warn':
                     self._display.warning(f'Skipping, did not find setting {term}')
                 elif missing != 'skip':
-                    raise AnsibleLookupError(f'Unable to find setting {term}') from e
+                    raise AnsibleLookupError(f'Unable to find setting {term}: {e}') from e
```

Comment:
if you do `from e` you will already get the message appended, this will make it so you get it x2

Reply:
closing this as we have not updated AnsibleError to handle `from e` yet ... i live in future as I'm working on PR that does this